### ISSUE TITLE
Audio node: filename can't be set dynamically

### ISSUE THREAD

**Issue by arneman** [guidone/node-red-contrib-chatbot#331]
in `chatbot-audio.js` change:
`var path = node.filename;`
to
`var path = node.filename || msg.payload;`

in `chatbot-audio.html` add behind `files<p>`:
```
<p>You can also set the filename with a function node:</p>
<pre><code>msg.payload = &#39;/my_dir/my_audio.png&#39;
</code></pre>
```

**Comment by guidone** [comment]
Thanks for the feedback, working on it

**Comment by guidone** [comment]
Hi @arneman which platform are you using?

**Comment by arneman** [comment]
Raspberry Pi 4, Rasbian Buster, nodered v 0.20.7

**Comment by guidone** [comment]
Sorry I meant which chat platform (facebook, telegram, etc)

**Comment by arneman** [comment]
Hehe, ok, and I was wondering why you are asking this ;)

I'm using telegram

Regards 

Arne

**Comment by guidone** [comment]
Just released `0.17.0`, all media nodes were revamped, look here for some [[(breaking changes)|Breaking-Changes-0.16.0]]**
